apibinding/examples/example_py_3_11.py

```
@@ -0,0 +1,87 @@
+
+    rsp = json.loads(rsp_body)
+
+    if rsp["state"] == "Done":
+        return json.loads(rsp["result"])
+
+    job_uuid = rsp["uuid"]
+    def query_until_done():
+        conn.request("GET", "/zstack/api/result/%s" % job_uuid)
+        response = conn.getresponse()
+        if response.status != 200:
+            raise Exception("failed to query API result, %s, %s" % (response.status, response.reason))
```

Ruff (0.15.4)

[warning] Create your own exception

(TRY002)